geniza/corpus/models.py

```diff
@@ -1219,10 +1229,6 @@ def index_data(self):
             "post_save": DocumentSignalHandlers.related_save,
             "pre_delete": DocumentSignalHandlers.related_delete,
         },
-        "tags": {
-            "post_save": DocumentSignalHandlers.related_save,
-            "pre_delete": DocumentSignalHandlers.related_delete,
-        },
```

Comment:
We still need these handlers, though, right? If someone renames a tag without changing any m2m relationships, we should reindex all the documents with that renamed tag so they reflect the update